index.js

```diff
@@ -104,6 +104,7 @@
 
   //  identity :: a -> a
   function identity(x) { return x; }
+  Object.freeze(identity);
```

Comment:
I'm anti-freeze too. What are you thinking could happen if we do not freeze this function, Gabe? Someone might attach properties to it? Would this even cause problems?

Currently we don't even freeze `Z` itself, and mutation such as `Z.lt = Z.gt;` could certainly cause problems in practice. ;)